File: Searching and Sorting Algorithms/17AccumulateCount.py
n=int(input())
arr=list(map(int,input().split()))
k=int(input())
res=[0]*k
for i in range(n):
    res[arr[i]]+=1
prev=0
for j in range(k):
    res[j]+=prev
    prev=res[j]

print(*res)

File: Searching and Sorting Algorithms/codigo_cs/17AccumulateCount.cs
using System;

public class Test17
{
    public static void AccumulateCount()
    {
        // your code goes here
        int n = int.Parse(Console.ReadLine());
        int[] arr = Array.ConvertAll(Console.ReadLine().Split(' '), int.Parse);
        int k = int.Parse(Console.ReadLine());
        int[] res = Enumerable.Repeat(0, k).ToArray();

        for (int i = 0; i < n; i++)
        {
            res[arr[i]]++;
        }
        int prev = 0;
        for (int j = 0; j < k; j++)
        {
            res[j] += prev;
            prev = res[j];
        }
        Console.WriteLine(string.Join(" ", res));
    }
}